mcs/class/System/Test/System.Net.NetworkInformation/IPInterfacePropertiesTest.cs

```diff
@@ -116,6 +116,8 @@ public void DnsEnabled ()
 		{
 			if (Environment.OSVersion.Platform == PlatformID.Win32NT)
 				Assert.Ignore ("IsDnsEnabled is not nessasarily enabled for all interfaces on windows.");
+			if (Environment.OSVersion.Platform != PlatformID.Win32NT)
+				Assert.Ignore ("IsDnsEnabled is not supported on Unix.");
```

Comment:
It seems corefx has an implementation for this. It might throw a PlatformNotSupportedException in some cases but we can catch that and use Assert.Inconclusive instead of essentially disabling the test everywhere.